File: MySqlTestInJava/src/MySqlTest/Person.java
package MySqlTest;

import java.sql.Date;

/**
 * Base class for Person.
 */
public class Person
{
    //<editor-fold defaultstate="collapsed" desc="Fields">
    
        // The id field of the Person class.
        private int id;

        // The name field of the Person class.
        private String name;

        // The mothername field of the Person class.
        private String mothername;

        // The location field of the Person class.
        private String location;

        // The birthdate field of the Person class.
        private Date birthdate;

        // The isDeleted field of the Person class.
        private boolean isDeleted;
    
    // </editor-fold>
    
    //<editor-fold defaultstate="collapsed" desc="Properties">
    
    /**
     * Gets the id.
     * 
     * @return The id.
     */
    public int getId()
    {
        return id;
    }

    /**
     * Sets the id.
     * 
     * @param id The id to set.
     */
    public void setId(int id)
    {
        this.id = id;
    }

    /**
     * Gets the name.
     * 
     * @return The name.
     */
    public String getName()
    {
        return name;
    }

    /**
     * Sets the name.
     * 
     * @param name The name to set.
     */
    public void setName(String name)
    {
        this.name = name;
    }

    /**
     * Gets the mothername.
     * 
     * @return The mothername.
     */
    public String getMothername()
    {
        return mothername;
    }

    /**
     * Sets the mothername.
     * 
     * @param mothername The mothername to set.
     */
    public void setMothername(String mothername)
    {
        this.mothername = mothername;
    }

    /**
     * Gets the location.
     * 
     * @return The location.
     */
    public String getLocation()
    {
        return location;
    }

    /**
     * Sets the location.
     * 
     * @param location The location to set.
     */
    public void setLocation(String location)
    {
        this.location = location;
    }

    /**
     * Gets the birthdate.
     * 
     * @return The birthdate.
     */
    public Date getBirthdate()
    {
        return birthdate;
    }

    /**
     * Sets the birthdate.
     * 
     * @param birthdate The birthdate to set.
     */
    public void setBirthdate(Date birthdate)
    {
        this.birthdate = birthdate;
    }

    /**
     * Gets the isDeleted.
     * 
     * @return The isDeleted.
     */
    public boolean getIsDeleted()
    {
        return isDeleted;
    }

    /**
     * Sets the isDeleted.
     * 
     * @param isDeleted The isDeleted to set.
     */
    public void setIsDeleted(boolean isDeleted)
    {
        this.isDeleted = isDeleted;
    }
    
    // </editor-fold>
    
    //<editor-fold defaultstate="collapsed" desc="Constructors">
    
    /**
     * Initializes a new instance of the Person class.
     */
    public Person()
    { }
    
    /**
     * Initializes a new instance of the Person class.
     * 
     * @param id The input value of the id field.
     * @param name The input value of the name field.
     * @param mothername The input value of the mothername field.
     * @param location The input value of the location field.
     * @param birthdate The input value of the birthDate field.
     * @param isdeleted The input value of the isDeleted field.
     */
    public Person(int id, String name, String mothername, String location, Date birthdate, boolean isdeleted)
    {
        this.id = id;
        this.name = name;
        this.mothername = mothername;
        this.location = location;
        this.birthdate = birthdate;
        this.isDeleted = isdeleted;
    }
    
    // </editor-fold>
    
    //<editor-fold defaultstate="collapsed" desc="Methods">
    
    // </editor-fold>
}

File: MySqlTestInPython/Person.py
# Base class for Person.
class Person(object):
    """description of class"""

    # region Fields

    # The id field of the Person class.
    __id = None

    # The name field of the Person class.
    __name = None

    # The mothername field of the Person class.
    __mothername = None

    # The location field of the Person class.
    __location = None

    # The birthdate field of the Person class.
    __birthdate = None

    # The isDeleted field of the Person class.
    __isDeleted = None

    # endregion Fields

    # region Properties

    # Gets the id.
    def getId(self):
        return self.__id

    # Sets the id.
    def setId(self, value):
        self.__id = value

    # Gets the name.
    def getName(self):
        return self.__name

    # Sets the name.
    def setName(self, value):
        self.__name = value

    # Gets the mothername.
    def getMothername(self):
        return self.__mothername

    # Sets the mothername.
    def setMothername(self, value):
        self.__mothername = value

    # Gets the location.
    def getLocation(self):
        return self.__location

    # Sets the location.
    def setLocation(self, value):
        self.__location = value

    # Gets the birthdate.
    def getBirthdate(self):
        return self.__birthdate

    # Sets the birthdate.
    def setBirthdatee(self, value):
        self.__birthdate = value

    # Gets the isDeleted.
    def getIsDeleted(self):
        return self.__isDeleted

    # Sets the isDeleted.
    def setIsDeleted(self, value):
        self.__isDeleted = value

    # endregion Properties

    # region Constructors

    # Initializes a new instance of the Person class.
    def __init__(self):
        self.__id = None
        self.__name = None
        self.__mothername = None
        self.__location = None
        self.__birthdate = None
        self.__isDeleted = None

    # Initializes a new instance of the Person class.
    def __init__(self, id, name, mothername, location, birthdate, isdeleted):
        self.__id = id
        self.__name = name
        self.__mothername = mothername
        self.__location = location
        self.__birthdate = birthdate
        self.__isDeleted = isdeleted

    # endregion Constructors

    # region Methods

    # endregion Methods